Makefile

```diff
@@ -133,8 +134,11 @@ test: all
 ############# LOCAL INSTALLATION AND DOWNSTREAM PACKAGING ##############
 #
 version: VERSION mods/ra/mod.yaml mods/cnc/mod.yaml mods/d2k/mod.yaml mods/ts/mod.yaml mods/modcontent/mod.yaml mods/all/mod.yaml
-	@sh -c '. ./packaging/functions.sh; set_engine_version $(VERSION) .'
-	@sh -c '. ./packaging/functions.sh; set_mod_version $(VERSION) mods/ra/mod.yaml mods/cnc/mod.yaml mods/d2k/mod.yaml mods/ts/mod.yaml mods/modcontent/mod.yaml mods/all/mod.yaml'
+ifeq ($(GIT_VERSION), git-)
+	$(error Unable to determine version from git repository)
+endif
+	@sh -c '. ./packaging/functions.sh; set_engine_version "$(GIT_VERSION)" .'
+	@sh -c '. ./packaging/functions.sh; set_mod_version "$(GIT_VERSION)" mods/ra/mod.yaml mods/cnc/mod.yaml mods/d2k/mod.yaml mods/ts/mod.yaml mods/modcontent/mod.yaml mods/all/mod.yaml'
```

Comment:
Why is this not using `VERSION` anymore? Does that even make sense when we error above if `GIT_VERSION` couldn't be set?

Reply:
> Why is this not using `VERSION` anymore?

The **version** target sets/updates the current version to `GIT_VERSION` (by modifying some files).
The new `VERSION` variable reads the currently set version (from one of these files).

> Does that even make sense when we error above if `GIT_VERSION` couldn't be set?

I think it's better to let `make version` fail when git is not available, instead of setting nonsense as the version.
I'm not quite getting your question wrong.

Reply:
Ah my bad I thought there was an `echo` up there but it just is `error`.